### PR TITLE
Update more common steps

### PR DESCRIPTION
**Pull request by ghost**
## Summary

This PR aims to test common steps that work first with the Volta project. There are some updates:
- Update more common steps
- Add feature files to test these common steps
- Update and adjust logic for methods in the `base.page.ts` file
## Types of Changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which does not change existing behavior or add new functionality)
- [ ] Library update (non-breaking change that will update one or more libraries to newer versions)
- [ ] Domestic (documentation, non-breaking change that doesn't change code behavior, can skip testing)

## Jira Issues:

https://c0x12c.atlassian.net/browse/TP-131

### CODE REVIEW

File: src/features/common.step.ts [hunk 1]
```diff
@@ -97,9 +173,9 @@ When('I click element with role {string} and name {string}', async ({ basePage }
   await basePage.clickByRole(role, name)
 })
 
-When('I click element with text {string}', async ({ basePage }, text) => {
-  await basePage.clickByText(text)
-})
+// When('I click element with text {string}', async ({ basePage }, text) => {
+//   await basePage.clickByText(text)
+// })
```
Comment (Boonichi) [inline src/features/common.step.ts]: nit: should keep this ?

File: src/features/common.step.ts [hunk 2]
```diff
@@ -175,27 +300,27 @@ Then('I expect that the element with role {string} and order {int} is visible',
...
+//   await expect(basePage.getPage().getByText(addressStr)).toBeVisible()
+// })
+
+// Then(
+//   'I expect that the address {string} with order {int} is invisible',
+//   async ({ commonDataProvider, basePage }, dataKey, order) => {
+//     const addressStr = shortenAddress(getDataByKey(commonDataProvider.commonData, dataKey))
+//     await expect(basePage.getPage().getByText(addressStr).nth(order)).toBeVisible()
+//   },
+// )
```
Comment (Boonichi) [inline src/features/common.step.ts]: nit: also this ?

**Review by Boonichi — APPROVED** [review]

File: src/features/common.step.ts (hunk 2 above)
Comment (ghost) [reply src/features/common.step.ts]: Sure, but I think from lines 312 to 323, it's not a common step that can be used for other projects, just only for Volta, right?

Comment (spartan-hungphan) [reply src/features/common.step.ts]: Yes, i think so. If you can't execute the code with them, it mean it's not common. Let's remove them

**Review by spartan-hungphan — APPROVED** [review]

**Pull request merged**